### PR TITLE
[MINOR][DOC] Add missing space after comma

### PR DESCRIPTION
**Pull request by attilapiros**
Adding missing spaces after commas.

### CODE REVIEW

File: docs/mllib-statistics.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Refer to the [`Statistics` Python docs](api/python/pyspark.mllib.html#pyspark.ml
 ### Streaming Significance Testing
 `spark.mllib` provides online implementations of some tests to support use cases
 like A/B testing. These tests may be performed on a Spark Streaming
-`DStream[(Boolean,Double)]` where the first element of each tuple
+`DStream[(Boolean, Double)]` where the first element of each tuple
```
Comment (srowen) [inline docs/mllib-statistics.md]: OK if we're bothering, search for `\[\(\w+,\w+\)\]`; there are some more like this

File: docs/configuration.md [hunk 2]
```diff
@@ -2296,7 +2296,7 @@ In some cases, you may want to avoid hard-coding certain configurations in a `Sp
 instance, Spark allows you to simply create an empty conf and set spark/spark hadoop properties.
 
 {% highlight scala %}
-val conf = new SparkConf().set("spark.hadoop.abc.def","xyz")
+val conf = new SparkConf().set("spark.hadoop.abc.def", "xyz")
```
Comment (srowen) [inline docs/configuration.md]: Likewise you might spot more of these with a crude regex like `","[^), .]`; I found a few more in .scala and .md files

**Review by srowen — CHANGES_REQUESTED** [review]

File: docs/mllib-statistics.md (hunk 1 above)
Comment (attilapiros) [reply docs/mllib-statistics.md]: I have only found one which is not a documentation link target (where having no space is the correct):
```
$ find . -name "*.md" -exec egrep "\[\(\w+\,\w+\)\]" \{} \;  -print
[GraphOps.joinVertices]: api/scala/index.html#org.apache.spark.graphx.GraphOps@joinVertices[U](RDD[(VertexId,U)])((VertexId,VD,U)⇒VD)(ClassTag[U]):Graph[VD,ED]
[Graph.outerJoinVertices]: api/scala/index.html#org.apache.spark.graphx.Graph@outerJoinVertices[U,VD2](RDD[(VertexId,U)])((VertexId,VD,Option[U])⇒VD2)(ClassTag[U],ClassTag[VD2]):Graph[VD2,ED]
[GraphOps.collectNeighbors]: api/scala/index.html#org.apache.spark.graphx.GraphOps@collectNeighbors(EdgeDirection):VertexRDD[Array[(VertexId,VD)]]
[GraphOps.pregel]: api/scala/index.html#org.apache.spark.graphx.GraphOps@pregel[A](A,Int,EdgeDirection)((VertexId,VD,A)⇒VD,(EdgeTriplet[VD,ED])⇒Iterator[(VertexId,A)],(A,A)⇒A)(ClassTag[A]):Graph[VD,ED]
[Graph.apply]: api/scala/index.html#org.apache.spark.graphx.Graph$@apply[VD,ED](RDD[(VertexId,VD)],RDD[Edge[ED]],VD)(ClassTag[VD],ClassTag[ED]):Graph[VD,ED]
[Graph.fromEdgeTuples]: api/scala/index.html#org.apache.spark.graphx.Graph$@fromEdgeTuples[VD](RDD[(VertexId,VertexId)],VD,Option[PartitionStrategy])(ClassTag[VD]):Graph[VD,Int]
      sendMsg: EdgeTriplet[VD, ED] => Iterator[(VertexId,A)],
./graphx-programming-guide.md
```

So the last one: `sendMsg: EdgeTriplet[VD, ED] => Iterator[(VertexId,A)],`. 
Which will be corrected in my next commit.

File: docs/configuration.md (hunk 2 above)
Comment (attilapiros) [reply docs/configuration.md]: I would like to keep this PR minor and only focusing on `.md` files.

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**